========== TEST 1 ==========
Default constructor + push_back
9 - 16
hello
hello
hello
hello
hello
hello
hello
hello
hello
Constructor with size
9 - 9









Constructor with size and value
9 - 9
hello
hello
hello
hello
hello
hello
hello
hello
hello
Constructor with two iterators
9 - 9
hello
hello
hello
hello
hello
hello
hello
hello
hello
Copy constructor
9 - 9
hello
hello
hello
hello
hello
hello
hello
hello
hello
========== TEST 2 ==========
reserve
5 - 9
salut
salut
salut
salut
salut
resize
8 - 18
salut
salut
salut
salut
salut
hello
hello
hello
assign iterator
8 - 8
salut
salut
salut
salut
salut
hello
hello
hello
assign value
12 - 12
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
ouais ouais ouais
========== TEST 3 ==========
erase
7 - 8
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
erase range
13 - 15
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
erase range same position
15 - 15
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
========== TEST 4 ==========
clear
0 - 8
swap (member)
15 - 15
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
15 - 15
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
wsh
swap (non-member)
15 - 15
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
yo man
15 - 15
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
YAX3
========== TEST 5 ==========
insert (1 element)
13 - 24
0
3
6
9
12
15
18
555
21
24
27
30
33
insert (range)
17 - 24
0
3
6
9
12
555
555
555
555
555
15
18
21
24
27
30
33
========== TEST 6 ==========
push_back (no realloc)
5 - 5
yo
yo
yo
yo
cc
push_back (realloc)
15 - 20
yo
yo
yo
yo
cc
yo
yo
yo
yo
yo
yo
yo
yo
yo
yo
pop_back (1 element)
14 - 20
yo
yo
yo
yo
cc
yo
yo
yo
yo
yo
yo
yo
yo
yo
pop_back (multiple elements)
6 - 20
yo
yo
yo
yo
cc
yo
========== TEST 7 ==========
--- ---
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (true)
+ >  OK (false)
+ <= OK (true)
+ >= OK (false)
--- ---
--- ---
+ == OK (true)
+ != OK (false)
+ <  OK (false)
+ >  OK (false)
+ <= OK (true)
+ >= OK (true)
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (false)
+ >  OK (true)
+ <= OK (false)
+ >= OK (true)
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (false)
+ >  OK (true)
+ <= OK (false)
+ >= OK (true)
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (true)
+ >  OK (false)
+ <= OK (true)
+ >= OK (false)
--- ---
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (true)
+ >  OK (false)
+ <= OK (true)
+ >= OK (false)
--- ---
+ == OK (false)
+ != OK (true)
+ <  OK (false)
+ >  OK (true)
+ <= OK (false)
+ >= OK (true)
--- ---
--- 2 - 2
========== TEST 8 ==========
assign / insert (gen par copilot)
15 - 20
1
1
2
2
3
3
3
3
3
2
2
2
1
1
1
